src/lib/Hydra/Controller/API.pm

```diff
@@ -285,6 +285,22 @@ sub push_github : Chained('api') PathPart('push-github') Args(0) {
     $c->response->body("");
 }
 
+sub push_gitea : Chained('api') PathPart('push-gitea') Args(0) {
+    my ($self, $c) = @_;
+
+    $c->{stash}->{json}->{jobsetsTriggered} = [];
+
+    my $in = $c->request->{data};
+    my $url = $in->{repository}->{clone_url} or die;
```

Comment:
```suggestion
    my $url = $in->{repository}->{clone_url} or die;
    $url =~ s/.git$//;
```

Reply:
Would that still match if the flake input had ``.git``?

Reply:
i would think so, since the flake/url matching is done with LIKE and wildcards before and after the URL